.mergify.yml

```diff
@@ -1,14 +1,4 @@
 pull_request_rules:
-  - name: Auto merge
-    conditions:
-      - "#changes-requested-reviews-by=0"
```

Comment:
Instead of "request changes", [disable auto-merge](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request#disabling-auto-merge) will be used (by anyone with write permission) in an emergency.